### PR TITLE
raidboss: Korean translate omega raids (savage)

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
I think there's no need to use `--Boss Targetable--` in o4s, just use `--Targetable--` to use common replacements.

### CODE REVIEW

**Comment by quisquous** [comment]
Sorry about this o4s timeline.  It's one of the very old ones that came from elsewhere.  I think this, the o3s timeline, and the a7s timeline are maybe the only ones left that haven't been redone at this point.

**Comment by Jaehyuk-Lee** [comment]
I checked o3s and a7s, I found strange timeline
```
267 ""
267 ""
267 ""
267 ""
267 ""
267 ""
267 ""
267 ""
267 ""
267 ""
```
I just removed them.

**Comment by quisquous** [comment]
Unfortunately, I think you should not remove those blank lines.  Those are there to add space around syncs.  In timelines made for cactbot, we move things forward in time to hide future mechanics.  In these older timelines, they added blank spaces to do that.

I think it's best to leave as-is until somebody remakes this timeline better.  This is why it says `timelineNeedsFixing: true,` in a7s.js.

**Comment by Jaehyuk-Lee** [comment]
okay

File: ui/raidboss/data/04-sb/raid/o8s.js [hunk 1]
```diff
@@ -843,26 +842,38 @@
       'replaceText': {
         'Aero Assault': '갈기갈기 에어로가',
         'All Things Ending': '소멸의 발차기',
+        'big': '大',
+        'small': '小',
```
Comment (MaikoTan) [inline ui/raidboss/data/04-sb/raid/o8s.js:846]: I am curious why here are 漢字 not Korean 諺文( If I am not wrong?)

Comment (Jaehyuk-Lee) [reply ui/raidboss/data/04-sb/raid/o8s.js:846]: There's very few Hanja that used commonly in Korea, 大's pronounciation is '대" but that's very uncommon to use separately, needs context for fast recognition, so people seeing that.. "hmmm what?"
I tried to use pure Korean word, '큰" but adjactive in bracket alone is also weird in Korean.
I'd write pure Korean version if those hanja was not common, but 大 and 小 is very well known in Korea.

**Comment by quisquous** [comment]
Sorry for causing merge conflicts here.  I can fix these and get this landed tomorrow.

**Comment by quisquous** [comment]
Thank you.  Sorry again about the merge conflicts in o4s.

**Pull request merged**